pvmismatch/contrib/xlsio/xlsio/xlsio.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+import numpy as np
+from pvmismatch.pvmismatch_lib import pvcell, pvconstants, pvmodule, pvstring, pvsystem
+
+def _create_cell_pos_df(pv_mod, nrows, nr_string, nr_mod):
+    """Create cell position dataframe of a module in the PV system"""
+    cell_pos = pv_mod.cell_pos
+    cell_pos_df = pd.DataFrame(index=['{}_{}'.format(nr_mod, nr) for nr in range(nrows)])
+    b = 0
+    for bypass in cell_pos:
+        c = 0
+        for col in bypass:
+            cell_pos_df['{}_{}_{}'.format(nr_string, b, c)] = [i['idx'] for i in col]
+            c = c+1
+        b = b+1
```

Comment:
you can use inplace addition for both lines:

    c += 1  # equivalent to c = c + 1
    b += 1  # equivalent to b = b + 1

Reply:
Thanks, I went for using enumerate where possible.